src/integrations/prefect-gcp/prefect_gcp/__init__.py

```diff
@@ -1,7 +1,3 @@
-from prefect._internal.compatibility.deprecated import (
```

Comment:
Same comment here.  I don't really mind these small changes mixed in with a larger PR, but if something happens and we need to revert, it could be nice to leave these other things in...

Reply:
I'm moving deprecation changes to https://github.com/PrefectHQ/prefect/pull/17632